### PR TITLE
[FIX] l10n_es_pos_tbai: Add refund reason in pos

### PR DESCRIPTION
**Pull request by robinengels**
Refund order in spain require a refund reason.

Steps to reproduce:
-------------------
* Install l10n_es_pos_tbai module
* Open PoS
* Make an order and try to refund it
> Observation: You get an error message saying that you need to add a
refund reason

opw-4282586

### CODE REVIEW

File: addons/l10n_es_pos_tbai/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from odoo import api, models
+
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    @api.model
+    def get_refund_reason_list(self):
+        return self.env['account.move']._fields['l10n_es_tbai_refund_reason'].selection
```
Comment (william-andre) [inline addons/l10n_es_pos_tbai/models/account_move.py:12]: You need the translated values
```suggestion
        return self.env['account.move']._fields['l10n_es_tbai_refund_reason']._description_selection(self.env)
```

**Comment by robinengels** [comment]
@robodoo up to 18.0

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by william-andre** [comment]
Why up to 18 though?

Always give the reason when you disable it. Otherwise it's better to let the forward port crash in the version where it's not needed anymore...

**Comment by robinengels** [comment]
Issue is already fixed in 18.0 indeed should have put it in my closing message

**Pull request closed without merging**